README.md

```diff
@@ -99,6 +99,7 @@ These derived images include a set of standard capabilities that enable many of
 - Python 2.7.12 and Python 3.5.2
 - Node.js 8.4.0
 - .NET Core SDK 2.0.0
+- Az CLI 2.0
 
```

Comment:
Can we use the full name "Azure CLI 2.0" here?

Reply:
done